client/src/Movies/MovieCard.js

```diff
@@ -1,7 +1,29 @@
 import React from 'react';
 
 const MovieCard = props => {
```

Comment:
Great work with MovieCard! The next step is to use MovieCard inside of MovieDetails and Movie.